server/app/views/admin/programs/ProgramPredicatesEditView.java

```diff
@@ -298,6 +312,45 @@ private static LabelTag renderPredicateQuestionCheckBoxRow(
                         .withClasses("mt-1", "text-sm")));
   }
 
+  private DivTag createEligibilityMessageForm(
+      Http.Request request, BlockDefinition block, ProgramDefinition program) {
+    String existingEligibilityMessage =
+        block.localizedEligibilityMessage().map(LocalizedStrings::getDefault).orElse("");
```

Comment:
**question:** (maybe blocking) Shouldn't we use [`LocalizedStrings.getOrDefault(Locale)`](https://github.com/civiform/civiform/blob/208acc61f31ad51beef3d073c7f39942c6366861/server/app/services/LocalizedStrings.java#L152) here so that we get the translated message?

Reply:
We wanna keep it in the default language English here because this page is for Civiform Admins.